lib/greenhouse_api/client.rb

```diff
@@ -27,6 +27,25 @@ def list_candidates(params = {})
       list_many('candidates', params)
     end
 
+    def get_current_offer_for_application(application_id)
+      response = request(
```

Comment:
The number of calls supported is still small so I think we can still manage in a single class. If this grows, we should look into abstract away these lower level logic

Reply:
agreed. i'd like to think about namespacing based on resources, for example. but agreed that it feels a little premature with just ~2 endpoints so far.

also i'll need your help to publish
```bash
bundle exec rake release
greenhouse_api 0.1.12 built to pkg/greenhouse_api-0.1.12.gem.
Tagged v0.1.12.
Pushed git commits and tags.
Pushing gem to https://rubygems.org...
You do not have permission to push to this gem. Ask an owner to add you with: gem owner greenhouse_api --add johnsontanner3@gmail.com
```